CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 8.4.3. (August 23, 2023)
+
+IMPROVEMENTS: 
+* resource/artifactory_permission_target: added 0 length check for `actions` block. Upgraded Terraform Plugin Framework to 1.3.5. 
```

Comment:
Let's have the upgrade text in separate bullet.

Reply:
Sure!